src/endpoint-core/Endpoint.Core/Response.Empty/EndpointEmptyResponseMetadataProvider.cs

```diff
@@ -0,0 +1,17 @@
+namespace PrimeFuncPack;
+
+public static class EndpointEmptyResponseMetadataProvider
+{
+    public static EndpointResponseMetadata ResponseMetadata { get; }
```

Comment:
Since the class/property is public, suggest changing the property to a method, since it produces new instances.
```suggestion
    public static EndpointResponseMetadata ResponseMetadata()
```

Or consider implementing a singleton.
(But I suppose producing new instances is more suitable here?)